subclass air_terminal_single_duct_vav_reheat_apply_minimum_damper_pos…

brianlball opened the pull request:
https://github.com/NREL/openstudio-bem-to-surrogate-gem/pull/215

brianlball committed:
subclass air_terminal_single_duct_vav_reheat_apply_minimum_damper_position()
and model_apply_multizone_vav_outdoor_air_sizing()

brianlball commented:
WIP until we resolve if we want to use https://github.com/NREL/openstudio-bem-to-surrogate-gem/pull/216 instead

jmarrec commented:
Seems like https://github.com/NREL/openstudio-bem-to-surrogate-gem/pull/216 was merged, wanna go ahead and close this one?

brianlball committed:
remove model_apply_multizone_vav_outdoor_air_sizing subclass

https://github.com/jmarrec/openstudio-standards/blob/179D/lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb#L970
